.github/workflows/ci.yml

```diff
@@ -54,7 +54,6 @@ jobs:
       - name: GolangCI-Lint
         if: github.event.name != 'pull_request' # See https://github.com/golangci/golangci-lint-action/issues/362
         run: |
-          curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.45.2
+          curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.52.2
```

Comment:
Looking through releases of [golangci/golangci-lint-action[(https://github.com/golangci/golangci-lint-action), it seems their latest release (3.6) should fix this issue. So, this hacky workaround might be not needed anymore.
However, this needs to be tested.

Reply:
From what I understand the workaround can be removed, I pushed a commit doing so. I have a test PR on my fork https://github.com/Earlopain/discordgo/pull/1 where you can look at some output.

I changed the trigger for push to only run on master because ci would run twice on PRs otherwise. They fall under both specified events. Previously the if checks on the steps took care of that.